types/pollyjs__persister-local-storage/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for @pollyjs/persister-local-storage 4.0
+// Project: https://github.com/netflix/pollyjs/tree/master/packages/@pollyjs/persister-local-storage
+// Definitions by: offirgolan <https://github.com/offirgolan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import Persister from '@pollyjs/persister';
+
+export default class LocalStoragePersister extends Persister {}
```

Comment:
Same here; [at runtime](https://runkit.com/embed/sqgcmm893xy7) the module doesn't use a default, but rather assigns to the module as a whole.